qdrant-landing/content/documentation/private-cloud/qdrant-cluster-management.md

```diff
@@ -287,4 +287,63 @@ step certificate create mydomain.com qdrant-nodes.crt qdrant-nodes.key \
   --san qdrant-my-cluster-0.qdrant-headless-my-cluster \
   --san qdrant-my-cluster-1.qdrant-headless-my-cluster
 ```
-</aside>
\ No newline at end of file
+</aside>
+
+## Creating a cluster with GPU support
```

Comment:
It would be nice if we can rename this to have better visibility in the TOC. Currently, it is not obvious that this entry is for GPU enabled cluster.
![image](https://github.com/user-attachments/assets/827cc80a-0445-4ccc-a2b9-75b4656afcd1)

Reply:
Done